From a63e12ecc13148c62a1b1a47ce65b9cfc4a2e1d9 Mon Sep 17 00:00:00 2001
From: Jehan <jehan@girinstud.io>
Date: Fri, 8 Jun 2018 16:38:32 +0200
Subject: [PATCH] desktop: removing all <release> description.

This is a test researching why appstream data fails to display on
flathub or GNOME Software.
With `appstream-util validate-strict`, I actually get a bunch of
"style-invalid" on the description, though it passes through with
`validate-relax`. Could it be it?

Let's revert this commit later.
---
 desktop/org.gimp.GIMP.appdata.xml.in.in | 144 +-----------------------
 1 file changed, 3 insertions(+), 141 deletions(-)

diff --git a/desktop/org.gimp.GIMP.appdata.xml.in.in b/desktop/org.gimp.GIMP.appdata.xml.in.in
index 57c47a0084..a1b182ad02 100644
--- a/desktop/org.gimp.GIMP.appdata.xml.in.in
+++ b/desktop/org.gimp.GIMP.appdata.xml.in.in
@@ -72,148 +72,10 @@
   </provides>
 
   <releases>
-    <release version="2.10.2" date="2018-05-20">
-      <description>
-        <_p>
-          This second release in the GIMP 2.10 series, so soon after
-          2.10.0, is mostly the usual bug-fixing version after a major
-          release, with a few dozen bugs fixed.
-        </_p>
-        <_p>
-          It also features a new plug-in for the support of the HEIF
-          format, both for importing and exporting, as well as 2 new
-          filters: "Spherize" and "Recursive Transform". These are
-          nice examples of our relaxed feature policy in stable micro
-          releases.
-        </_p>
-      </description>
-    </release>
+    <release version="2.10.2" date="2018-05-20"/>
+    <release version="2.10.0" date="2018-04-27"/>
 
-    <release version="2.10.0" date="2018-04-27">
-      <description>
-        <_p>
-          First release of the 2.10 series which prominently features
-          the port to a new image processing engine, GEGL.
-          The most outstanding changes are:
-        </_p>
-        <ul>
-          <_li>
-            High bit depth color processing (16/32-bit per color channel)
-          </_li>
-          <_li>
-	    Color management is a core feature now, most widgets
-	    and preview areas are color-managed
-          </_li>
-          <_li>
-	    On-canvas effect preview, with split view for before/after
-	    processing pixels
-          </_li>
-          <_li>
-            Multi-threaded and hardware-accelerated rendering,
-            processing and painting
-          </_li>
-          <_li>
-            Most tools improved, several new transformation tools
-          </_li>
-          <_li>
-	    Improved support for many image formats, in particular better
-	    PSD importing
-          </_li>
-          <_li>
-            Newly supported image formats: OpenEXR, RGBE, WebP, HGT…
-          </_li>
-          <_li>
-            Improved digital painting: canvas rotation and
-            flipping, symmetry painting, MyPaint brushes…
-          </_li>
-          <_li>
-	    Metadata viewing and editing for Exif, XMP, IPTC, and DICOM
-          </_li>
-          <_li>
-            Basic HiDPI support: automatically or user-selected icon size
-          </_li>
-          <_li>
-            New themes for GIMP: Light, Gray, Dark, and System
-          </_li>
-          <_li>
-            And much, much more…
-          </_li>
-        </ul>
-        <_p>For more information, see https://www.gimp.org/news/2018/04/27/gimp-2-10-0-released/</_p>
-      </description>
-    </release>
-
-    <release version="2.10.0~rc2" date="2018-04-17" type="development">
-      <description>
-        <_p>
-        In this second release candidate before GIMP 2.10.0, while
-        debugging is still a prime target, a new focus has been put on
-        speed and optimization in order to provide a smoother painting
-        experience. Bigger changes are:
-        </_p>
-        <ul>
-          <_li>Major core optimizations for painting and display,
-          including parallelized painting code.</_li>
-          <_li>Symmetries are now preserved in XCF files (saved as image
-          parasites).</_li>
-          <_li>"Light" and "Dark" themes rewritten from scratch to get
-          rid of various usability issues. "Lighter" and "Darker" themes
-          removed.</_li>
-          <_li> New GimpToolGyroscope on-canvas control, currently used
-          for the Panorama Projection filter. The widget provides
-          on-canvas interaction for 3D rotation (yaw, pitch, roll).</_li>
-          <_li>Plug-in debugging improved to output stack traces from
-          plug-ins with --stack-trace-mode command line option not only
-          on receiving signals but also on warnings and critical errors
-          when "fatal-warnings" debug key is set.</_li>
-        </ul>
-        <_p>For more information, see https://www.gimp.org/news/2018/04/17/gimp-2-10-0-rc2-released/</_p>
-      </description>
-    </release>
-    <release version="2.10.0~rc1" date="2018-03-26" type="development">
-      <description>
-        <_p>
-        GIMP 2.10.0-RC1 is the first release candidate before GIMP
-        2.10.0 stable release, with a focus on debugging and stability.
-        Other than the many bug fixes, most notable improvements are:
-        </_p>
-        <ul>
-          <_li>New dashboard dockable to monitor GIMP resource usage</_li>
-          <_li>New debug dialog to produce back traces and other debug
-          data, encouraging to report bugs</_li>
-          <_li>Unsaved images can now be recovered after a crash</_li>
-          <_li>Layer masks on layer groups</_li>
-          <_li>JPEG 2000 support improved for high bit depth and various
-          color spaces</_li>
-          <_li>Screenshot and color picking improved on various platforms</_li>
-          <_li>Metadata defaults preferences now available</_li>
-          <_li>Various GUI polishing</_li>
-        </ul>
-        <_p>For more information, see https://www.gimp.org/news/2018/03/26/gimp-2-10-0-rc1-released/</_p>
-      </description>
-    </release>
-
-    <release version="2.9.8" date="2017-12-12" type="development">
-      <description>
-        <_p>GIMP 2.9.8 introduces on-canvas gradient
-        editing and various enhancements while focusing on bugfixing
-        and stability.</_p>
-        <ul>
-          <_li>On-canvas gradient editing</_li>
-          <_li>Notification when an image is over/underexposed</_li>
-          <_li>Better and faster color management</_li>
-          <_li>Support for color picker and screenshots in Wayland on KDE Plasma</_li>
-          <_li>Paste in place feature</_li>
-          <_li>Many usability improvements</_li>
-          <_li>Manual can be displayed in the user's preferred language</_li>
-          <_li>Improvements for the Wavelet Decompose filter</_li>
-          <_li>Improved compatibility with Photoshop .psd files</_li>
-          <_li>New support for password-protected PDF</_li>
-          <_li>New support for HGT format (Digital Elevation Model data)</_li>
-        </ul>
-        <_p>For more information, see https://www.gimp.org/news/2017/12/12/gimp-2-9-8-released/</_p>
-      </description>
-    </release>
+    <release version="2.9.8" date="2017-12-12" type="development"/>
     <release version="2.9.6" date="2017-08-24" type="development" />
     <release version="2.9.4" date="2016-07-13" type="development" />
     <release version="2.9.2" date="2015-11-27" type="development" />
-- 
2.17.1

